skore/src/skore/item/cross_validation_item.py

```diff
@@ -152,7 +153,11 @@ def factory(cls, reporter):
             "hash": _hash_numpy(X_array),
         }
 
-        plot_bytes = plotly.io.to_json(plot, engine="json").encode("utf-8")
+        # FIXME: Maybe this logic belongs in CrossValidationPlots
+        plots_bytes = {
+            plot_name: plotly.io.to_json(plot, engine="json").encode("utf-8")
+            for plot_name, plot in dataclasses.asdict(plots).items()
```

Comment:
Could the plot name be "humanized" ?
`timing_normalized` => _Normalized timings_ ?

Reply:
You mean just in the front-end right? I mean, in their Python code user would still have to write
```python
reporter.plots.timing_normalized
```

Reply:
Yes only frontend side !

Reply:
it completely makes sense! I like that it's easy from the human name to find how to call it in python. With that logic, maybe we could even switch from `compare_scores` to just `scores"?

Reply:
Done (also renamed to `scores`)